.github/workflows/check-npm-task.yml

```diff
@@ -73,8 +73,9 @@ jobs:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           version: 3.x
 
-      - name: Validate package.json
-        run: task --silent npm:validate
+      # Temprary disable as ajv seems to have problems
+      #- name: Validate package.json
+      #  run: task --silent npm:validate
```

Comment:
There is now a fix for the problem with the `npm:validate` task in the upstream "template":

https://github.com/arduino/tooling-project-assets/commit/cc6f87c1bf8cb188d5b4f258900531c4e67d301b